src/Compilers/Test/Utilities/VisualBasic/CompilationTestUtils.vb

```diff
@@ -923,18 +926,18 @@ Friend Module CompilationUtils
         Dim expectedReader = New StringReader(expected)
         Dim actualReader = New StringReader(actual)
 
-        Dim expectedBuilder As New StringBuilder()
-        Dim actualBuilder As New StringBuilder()
+        Dim expectedBuilder = _PooledStringBuilderPool.Allocate()
+        Dim actualBuilder = _PooledStringBuilderPool.Allocate()
         Dim expectedLine = expectedReader.ReadLine()
         Dim actualLine = actualReader.ReadLine()
 
         While expectedLine IsNot Nothing AndAlso actualLine IsNot Nothing
             If Not expectedLine.Equals(actualLine) Then
-                expectedBuilder.AppendLine("<! " & expectedLine)
-                actualBuilder.AppendLine("!> " & actualLine)
+                expectedBuilder.Builder.AppendLine("<! " & expectedLine)
```

Comment:
>expectedBuilder.Builder [](start = 16, length = 23)

Will there be an advantage of caching ```expectedBuilder.Builder``` and ```actualBuilder.Builder``` in locals and reusing them throughout the function?